mozregression/utils.py

```diff
@@ -60,6 +61,15 @@ def names(self):
 CACHE_SESSION = None
 
 
+def retry_get(**kwargs):
```

Comment:
Yes, that's the main argument for the get method. for example when you do:
retry_get(args=(chset_url,))
this is the "chset_ur". The idea is to be able to simply call:
retry_get(chset_url)